### ISSUE TITLE
Missing keyboard navigation (via Tab) in UIX canvasses like the scene inspector

### ISSUE THREAD

**Issue by ucceah** [Neos-Metaverse/NeosPublic#2302]
## Is your feature request related to a problem? Please describe.
It's a pretty annoying, but (propably) easily mitigated flaw in desktop mode, that i cant use Tab to navigate, when tweaking parameters in the inspector tool. necessitating needlessly tedious use of the mouse when entering data.

## Relevant issues
Obviously his applies to all UIX canvases, as well as the login, where tab and enter dont work.

## Describe the solution you'd like
allow using tab/shift+tab to move between fields.

## Describe alternatives you've considered
i tried using the arrow keys in combination with various modifiers.

**Comment by Frooxius** [comment]
This  is planned feature, but it's not actually that easy to implement to properly, since the system needs to be able to figure out which element to switch to, which is often based on their spatial arrangement. It will be added at some point though, but we need to settle on the technical design first.

**Comment by ucceah** [comment]
I see, and thank you for the prompt response. I thought there must be a reason, why something this obvious and convenient wasnt in place already. From my laymans perspective, the obvious solution would be to just number the fields and checkboxes, but propably the reality under the hood is a lot more complicated than that.
